.github/workflows/closed-issue-message.yml

```diff
@@ -0,0 +1,19 @@
+name: Closed Issue Message
+on:
+  issues:
+    types: [closed]
+jobs:
+  auto_comment:
+    permissions:
+      issues: write
+    runs-on: ubuntu-latest
+    steps:
+    - uses: aws-actions/closed-issue-message@v1
+      with:
+        # These inputs are both required
+        repo-token: "${{ secrets.GITHUB_TOKEN }}"
+        message: |
+          ### ⚠️COMMENT VISIBILITY WARNING⚠️ 
```

Comment:
I know we use this on CDK but honestly I think the allcaps on this message might be a little too much. Up to you though.

Reply:
you're right this is pretty extreme lol